css/colourpicker.css

```diff
@@ -79,12 +108,19 @@
   width: 150px;
   height: 150px;
   cursor: crosshair;
-  background: linear-gradient(to right,#FFF,rgba(255,255,255,0));
 }
 .colourpicker .colourpicker-grid-inner {
   width: 100%;
   height: 100%;
+  background: linear-gradient(to right,#FFF,rgba(255,255,255,0));
+  filter:progid:DXImageTransform.Microsoft.gradient(gradientType=1,startColorstr=#FFFFFFFF,endColorstr=#00000000);
+}
+
+.colourpicker .colourpicker-grid-inner-2 {
+  width:100%;
```

Comment:
suggestions for a good beautifier? Command-line preferred but I suppose I can use a web tool if I must. Maybe there's a sublime add in...